releases/latest/beta-instanton/Dockerfile.ubi.openjdk17

```diff
@@ -82,7 +82,7 @@ ENV LANG='en_US.UTF-8' LANGUAGE='en_US:en' LC_ALL='en_US.UTF-8'
     rm -rf /tmp/openjdk.tar.xz;
 
 ENV JAVA_HOME=/opt/java/openjdk \
-    PATH="/opt/java/openjdk/bin:$PATH"
+    PATH="$PATH:/opt/java/openjdk/bin"
```

Comment:
@tjwatson FYI this PATH declaration in beta-instanton was also updated to append the JDK's bin directory instead of pretending it.